docs/sources/dashboards/create-reports/report-settings/index.md

```diff
@@ -0,0 +1,102 @@
+---
+keywords:
+  - grafana
+  - reporting
+  - settings
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Settings
+title: Reporting settings
+description: Manage organizational Reporting settings
+weight: 700
+refs:
+  grafana-enterprise:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+  configuration:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#filters
```

Comment:
Shouldn't this link point to `configure-grafana` directly? I mean, remove `#filters`

Reply:
I think the issue is that we should have two refs: 
- configuration-filters: that keeps `#filters` and is referenced line 97
- configuration-reporting: with destination set to `/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/enterprise-configuration/#reporting` and is referenced line 66

Reply:
I think the issue is that we should have two refs: 
- configuration-filters: that keeps `#filters` and is referenced line 97
- configuration-reporting: with destination set to `/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/enterprise-configuration/#reporting`, is referenced line 66